### PR TITLE
Import donations command

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

File: src/Entity/Donation.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ class Donation implements GeoPointInterface
     private $gender;
 
     /**
-     * @ORM\Column
+     * @ORM\Column(nullable=true)
```
Comment (ottaviano) [inline src/Entity/Donation.php]: need to change typehint of return type: 
```php
function getEmailAddress(): ?string
```

File: migrations/Version20190919134908.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        $this->addSql('ALTER TABLE donators DROP INDEX donator_unique_matching, ADD INDEX IDX_A902FDD7B08E074EA9D1C132C808BA5A (email_address, first_name, last_name)');
+        $this->addSql('ALTER TABLE donators CHANGE email_address email_address VARCHAR(255) DEFAULT NULL');
+        $this->addSql('ALTER TABLE donations ADD type VARCHAR(255) DEFAULT NULL, CHANGE email_address email_address VARCHAR(255) DEFAULT NULL');
+        $this->addSQL('UPDATE donations SET type = \'cb\'');
+        $this->addSql('ALTER TABLE donations CHANGE type type VARCHAR(255) NOT NULL');
+    }
+
+    public function down(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE donation_transactions ADD type VARCHAR(255) NOT NULL COLLATE utf8_unicode_ci');
```
Comment (ottaviano) [inline migrations/Version20190919134908.php]: `ADD type VARCHAR(255) NOT NULL DEFAULT 'cb' `?

File: src/Entity/Transaction.php [hunk 3]
```diff
@@ -99,22 +88,24 @@ class Transaction
      */
     private $donation;
 
-    public function __construct(Donation $donation, array $payboxPayload)
+    public function __construct(Donation $donation, ?array $payboxPayload, ?\DateTimeInterface $donatedAt = null)
```
Comment (ottaviano) [inline src/Entity/Transaction.php]: can we avoid changing the construct of Transaction if you don't use anymore Transaction for Transfer/Check payment?

File: migrations/Version20190919134908.php (hunk 2 above)
Comment (Remg) [reply migrations/Version20190919134908.php]: i dont want a "default" value in the end.

**Pull request closed without merging**